mise.toml

```diff
@@ -14,7 +14,7 @@ k3d = "5.8.1"
 kind = "0.24.0"
 kubectl = "1.32.6"
 kube-linter = "0.6.8"
-protoc = "3.20.0"
+protoc = "32.1"
```

Comment:
this isn't such a high bump it's only change in versioning, this is actually 3.21 using old semantics